src/validations/actionSchema.ts

```diff
@@ -1,7 +1,23 @@
 import * as Yup from 'yup';
 
-const actionSchema = Yup.object().shape({
+export const actionSchema = Yup.object().shape({
   feedback: Yup.string().required('Please fill out email')
 });
 
-export default actionSchema;
+export const lifecycleIdSchema = Yup.object().shape({
+  lifecycleId: Yup.string()
+    .nullable()
+    .length(7),
+  expirationDateDay: Yup.string()
+    .length(2)
+    .required('Please include a day'),
+  expirationDateMonth: Yup.string()
+    .length(2)
+    .required('Please include a month'),
+  expirationDateYear: Yup.string()
+    .length(4)
+    .required('Please include a year'),
+  scope: Yup.string().required('Please include a scope'),
```

Comment:
I dunno if we want to do this now, but should we put these validation errors in a i18n file?

Reply:
I was thinking about that, too. Let's confer with Chris when he's back.